lib/roo_on_rails/sidekiq/queue_latency.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_support/core_ext/numeric'
+require_relative './settings'
```

Comment:
We've been trying to _not_ use `require_relative` anywhere in this codebase, so far. Mind being explicit with the path?